src/db/tables/onerep_scans.js

```diff
@@ -0,0 +1,82 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+ import initKnex from 'knex'
+ import knexConfig from '../knexfile.js'
+ const knex = initKnex(knexConfig)
+ 
+/**
+ * @param {number} onerepProfileId
+ * @param {number} oneRepScanId
+ * @returns {Promise<object>} oneRepScanResults
```

Comment:
I think this and other references to the unspecific `object` type in this file can also instead refer to `ScanResult` from `onerep.ts` directly?